src/printer/index.js

```diff
@@ -333,60 +333,41 @@ function printWithItem(path, print) {
 }
 
 function printIf(path, print) {
-  function printOrElse(path) {
-    const n = path.getValue();
-
-    if (n.orelse && n.orelse.length > 0) {
-      if (n.orelse[0].ast_type === "If") {
-        return concat(path.map(printElseBody, "orelse"));
-      }
-      return concat([
-        hardline,
-        "else:",
-        concat(path.map(printElseBody, "orelse"))
-      ]);
-    }
-  }
-
-  function printElseBody(path) {
-    const n = path.getValue();
-
-    const parts = [];
-
-    if (n.ast_type === "If") {
-      parts.push(
-        concat([
-          hardline,
-          "elif ",
-          path.call(print, "test"),
-          ":",
-          indent(concat([hardline, printBody(path, print)]))
-        ])
-      );
-    } else {
-      parts.push(indent(concat([hardline, path.call(print)])));
-    }
-
-    const orElse = printOrElse(path);
+  const parent = path.getParentNode();
+  const n = path.getValue();
 
-    if (orElse) {
-      parts.push(orElse);
-    }
+  let type = "if";
 
-    return concat(parts);
+  if (
+    parent.ast_type === "If" &&
+    parent.orelse.length > 0 &&
```

Comment:
I think we'll want to check if `length === 1` here to handle cases like

```python
if foo:
    return 1
else:
    if bar:
        bar()
    return 2
```
right? In any case, it would be nice to have a test case like this (an `else` with an `if` and something else after the `if`).